templates/gallery/partials/map.snip.html

```diff
@@ -0,0 +1,43 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#map}}
+  <section class="gallery-component-map gallery-base-section gallery-base-rich-text clearfix">
+    <div class="absolute left-0 right-0 z1">
+      <div class="gallery-base-container mx-auto">
+        <h2 class="gallery-component-map-heading mb2">
+          {{{heading}}}
+        </h2>
+        <div class="flex items-center">
+          <h3 class="m0">{{{subheading}}}</h3>
+          {{> ./button.snip.html}}
+        </div>
+      </div>
+    </div>
+    <div class="gallery-component-map-frame">
+      <amp-iframe
+        height="300"
+        layout="fixed-height"
+        sandbox="allow-scripts allow-top-navigation"
+        scrolling="no"
+        frameborder="0"
+        src="gallery-map.html?marker={{marker.lat}},{{marker.lng}}"
```

Comment:
We might run into same origin issues if we do this in preview mode, prefix this with https://ampstart-iframes.firebaseapp.com/ (and the full path in dist)

Reply:
I'll do this, but I'm not sure how to do this so that it works on `localhost` dev environments as well as production. Is there any reference code I can look at?

Reply:
It wont work on localhost. It is a problem that ill have to fix :). For now we will need to make it work on staging and production. I can push to that domain once this PR is in and localhost would look good